UnityClient/Assets/Shaders/Holograms/EmmisiveStatic.shader

```diff
@@ -0,0 +1,82 @@
+﻿    Shader "Holograms/EmissiveStatic"{
+     
+        Properties {
+            _Color ("Color", Color)=(1,1,1,1)
+            _Glossiness ("Smoothness", Range(0,1))=0.5
+            _Metallic ("Metallic", Range(0,1))=0.0
+            _GIAlbedoColor ("Color Albedo (GI)", Color)=(1,1,1,1)
+            _GIAlbedoColorMultiplier ("Intensity (GI)", Float)= 1.0
+        }
+     
+        SubShader {
+        // ------------------------------------------------------------------
+        // Extracts information for lightmapping, GI (emission, albedo, ...)
+        // This pass it not used during regular rendering.
+            Pass
+            {
+                Name "META"
+                Tags {"LightMode"="Meta"}
+                Cull Off
+                CGPROGRAM
+                #pragma vertex vert_meta
+                #pragma fragment frag_meta2

```

Comment:
Why frag_meta2 here instead of just frag_meta?

Reply:
Will double check.

Reply:
So it appears by defining it as `frag_meta` it conflicts with some hidden Unity functionality as I got some very nasty errors about redefinition, lol. Resolving.